### PR TITLE
fixes #1844

### LINKED ISSUE #1844: Fix the in memory database config.
"DiagnosticMode": "MemDb" was set as defult for fast sync and beam sync.

### PR DESCRIPTION
**Pull request by tkstanczak**
Removes the accidental default diagnostic mode

### CODE REVIEW

File: src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs [hunk 1]
```diff
@@ -333,6 +333,55 @@ public void Stays_on_full_sync(string configWildcard, bool stickToFullSyncAfterF
...
+                ConfigProvider configProvider = GetConfigProviderFromFile(configFile);
+                IInitConfig initConfig = configProvider.GetConfig<IInitConfig>();
+                initConfig.DiagnosticMode.Should().Be(DiagnosticMode.None, configFile);
+            }
+        }
+
+        [Test]
+        public void Migrations_are_not_enabled_by_default()
+        {
+            foreach (string configFile in Resolve("*"))
```
Comment (SebastianDremo) [inline src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs:350]: take this `"*"` string to `TestCase` like u did later to leave no room for interpretation what it is

File: src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs [hunk 2]
```diff
@@ -333,6 +333,55 @@ public void Stays_on_full_sync(string configWildcard, bool stickToFullSyncAfterF
             }
         }
 
+        [Test]
+        public void Diagnostics_mode_is_not_enabled_by_default()
+        {
+            foreach (string configFile in Resolve("^spaceneth.cfg"))
```
Comment (SebastianDremo) [inline src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs:339]: same as later comment

File: src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs (hunk 1 above)
Comment (tkstanczak) [reply src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs]: fixed

File: src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs (hunk 2 above)
Comment (tkstanczak) [reply src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs]: fixed

**Pull request merged**